### DISCUSSION TITLE
Running unit tests in IntelliJ

### DISCUSSION THREAD

**Discussion by tahirali-csc** [cockroachdb/cockroach#92100] [Q&A] [answered]
I am trying to debug unit tests from IntelliJ but I keep getting following issues
```bash
# github.com/cockroachdb/cockroach/pkg/sql/lexbase
../../sql/lexbase/predicates.go:45:17: undefined: reservedKeywords
```

Any suggestions?

**Comment by jordanlewis** [comment, marked as answer]
You need to run `./dev generate` in order to build all required generated Go code. Building the database with `./dev` doesn't need to do this because Bazel maintains the generated Go code itself, in caches - but the IDE doesn't know where to look to get at the caches. So you need to manually run `./dev generate` (or `./dev generate go`, which should just build the Go code only) in order to get the IDE to see everything.

https://wiki.crdb.io/wiki/spaces/CRDB/pages/2221703221/Developing+with+Bazel This page has some more detail. Good luck and feel free to join #contributors channel in Community Slack to chat with other contributors.

https://www.cockroachlabs.com/join-community/
